docs/index.rst

```diff
@@ -27,6 +27,69 @@ Run ``lira``:
    # or
    python -m lira
 
+
+Development Install
```

Comment:
This section is the same as the one in docs/development.rst. What we need is an installation guide, using pip, from source (link to the development page here) and using pipx.